### PR TITLE
[12.0][MIG] maintenance_equipment_tags

### PR DESCRIPTION
**Pull request by jarroyomorales**
Standard migration, no changes.

@etobella  @NuriaMForgeFlow  @dalonsod

### CODE REVIEW

**Review by etobella — APPROVED** [review]
LGTM :+1: No major changes

File: maintenance_equipment_tags/views/maintenance_equipment_tag.xml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+<odoo>
+
+    <record id="maintenance_equipment_tag_form_view" model="ir.ui.view">
+        <field name="name">maintenance.equipment.tag.form (in maintenance_equipment_tags)</field>
+        <field name="model">maintenance.equipment.tag</field>
+        <field name="arch" type="xml">
+            <form>
+                <sheet>
+                    <group>
```
Comment (dalonsod) [inline maintenance_equipment_tags/views/maintenance_equipment_tag.xml]: Consider add a `name` attribute for further extensions of this view

**Review by dalonsod — CHANGES_REQUESTED** [review]
See comment.

The module has no demo data, could you add it?

Also functionally tested, LGTM

**Review by NuriaXifre — APPROVED** [review]
LGTM! Functional review.

**Comment by jarroyomorales** [comment]
@dalonsod Comments attended!

**Review by dalonsod — APPROVED** [review]
👍

**Review by etobella — CHANGES_REQUESTED** [review]
Check version

**Review by etobella — APPROVED** [review]

**Comment by etobella** [comment]
/ocabot merge
Now the version is fine :wink:

**Pull request merged**